guides/common/modules/proc_returning-multiple-pages.adoc

```diff
@@ -0,0 +1,16 @@
+[id="returning-multiple-pages"]
+= Returning multiple pages
+
+You can use a `for` loop structure to get multiple pages of results.
+This example returns pages 1 to 3 of Content Views with 5 results per page.
+
+.Bash script
```

Comment:
Should I rename this?

Reply:
Well, the following code is a bash script. Perhaps "Bash script example"? Or did you have something else in mind? I'm not sure renaming is necessary.

Reply:
It _is_ a Bash script, so yes. I think we can omit "example" due to the introduction in line 5.